Dockerfile

```diff
@@ -38,8 +39,43 @@ RUN \
   apt-get remove -y git && \
   apt-get -y autoremove
 
+COPY . ./
+
 USER node
 ENV NODE_ENV=production
 
+ENV MQTT_HOST=localhost \
+    MQTT_PORT=1883 \ 
+    MQTT_QOS=1 \
+    MQTT_RETAIN=false
+
+ENV AMQP_HOST=localhost \
```

Comment:
Does a default value of localhost make sense? I would assume that  the Docker Image should only have one concern and that the MQTT server, the AMQP server etc. should be separate containers. If this is the case the the values `localhost` would never be correct.